src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkRepositoryModule.java

```diff
@@ -79,7 +79,6 @@ public StarlarkCallable repositoryRule(
     builder.addAttribute(attr("$configure", BOOLEAN).defaultValue(configure).build());
     if (thread.getSemantics().experimentalRepoRemoteExec()) {
       builder.addAttribute(attr("$remotable", BOOLEAN).defaultValue(remotable).build());
-      BaseRuleClasses.execPropertiesAttribute(builder);
```

Comment:
@juliexxia Is there a reason why repository rules have `exec_properties`? They can't be run remotely and are outside the normal action system, so it seems like it'd be unused. Might be simpler to just remove it from repo rules entirely rather than adding this conditionally.

Reply:
It looks like @buchgr added this in  https://github.com/bazelbuild/bazel/commit/755e29dfb21d7abfc2bab8dc30813fd5e340d930 with the note "This change is in preparation for adding remote execution capabilities
to repository_context.execute()." Maybe @buchgr has more context?